ElectoralRegisterResidentInformationApi/V1/Controllers/ElectoralRegisterResidentInformationApiController.cs

```diff
@@ -1,21 +1,21 @@
-using BaseApi.V1.Boundary.Response;
-using BaseApi.V1.UseCase.Interfaces;
+using ElectoralRegisterResidentInformationApi.V1.Boundary.Response;
+using ElectoralRegisterResidentInformationApi.V1.UseCase.Interfaces;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
-namespace BaseApi.V1.Controllers
+namespace ElectoralRegisterResidentInformationApi.V1.Controllers
 {
     [ApiController]
     //TODO: Rename to match the APIs endpoint
     [Route("api/v1/residents")]
     [Produces("application/json")]
     [ApiVersion("1.0")]
     //TODO: rename class to match the API name
-    public class BaseApiController : BaseController
+    public class ElectoralRegisterResidentInformationApiController : BaseController
```

Comment:
Could call this `ElectoralRegisterController` and remove `ResidentInformationApi`?